src/components/AppLayout/AppChannelContext.tsx

```diff
@@ -54,6 +61,10 @@ export const AppChannelProvider: React.FC = ({ children }) => {
     }
   }, [channelData]);
 
+  React.useEffect(() => {
+    onChannelChange(selectedChannel);
```

Comment:
This method should ideally be accessible from a SDK hook. Something along the lines of:
```
const { channel, setChannel } = useSaleorConfig();
```